### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by Alaboudi1**

### CODE REVIEW

File: src/components/main/App.tsx [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/*global chrome*/
+/*src/components/main/App.tsx*/
```
Comment (Alaboudi1) [inline src/components/main/App.tsx:2]: What does this comment mean? I know the global tell typescript that this is a global variable and needs not to worry about the declaration. But what the /*src/components/main/App.tsx*/ means?

Comment (saisree27) [reply src/components/main/App.tsx:2]: So I was looking some documentation for how content scripts work, and I saw that there was a comment at the top which specified the path of the file. Originally, I thought that this was necessary for content scripts but I saw that it didn't make any difference. I don't believe it does anything, but I can remove the comment in my code and make a new pull request.

Comment (Alaboudi1) [reply src/components/main/App.tsx:2]: sure, remove only the second line and push your changes. You do not need to make a new pull request.

**Comment by Alaboudi1** [comment]
Nice work getting your first pull request!! Cannot wait to see many pull requests coming from you :)

**Pull request merged**